worker/httpserver/manifold.go

```diff
@@ -4,6 +4,7 @@
 package httpserver
 
 import (
+	stdcontext "context"
```

Comment:
Why the alias?

Reply:
dependency.Context is always a pain here.